openmetadata-ui/src/main/resources/ui/src/styles/components/radio.less

```diff
@@ -36,3 +36,52 @@
     }
   }
 }
+
+// Global styling for segment toggle classs
+.segment-toggle {
+  background: @white;
+  border-radius: @size-sm;
+  border: 1px solid @grey-20;
+
+  .ant-segmented-item {
+    color: @text-color;
+    transition: all 0.3s ease;
+
+    &-selected {
+      background: @blue-23;
+      color: @blue-9;
+      text-align: center;
+      border-radius: @size-sm;
```

Comment:
The border-radius for the selected-toggle should be 8px ig. Can you try 8px here
<img width="60" alt="Screenshot 2025-04-09 at 1 09 55 PM" src="https://github.com/user-attachments/assets/8b44994a-cc69-4f73-a3b0-3e34243043b3" />

For eg, it should look like this
<img width="313" alt="Screenshot 2025-04-09 at 1 12 10 PM" src="https://github.com/user-attachments/assets/ec13de39-0571-472c-9c0e-c6b011008f3e" />